2022-03-13 21:31:20,114 INFO    MainThread:37301 [wandb_setup.py:_flush():75] Loading settings from /nethome/mrungta8/.config/wandb/settings
2022-03-13 21:31:20,114 INFO    MainThread:37301 [wandb_setup.py:_flush():75] Loading settings from /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning/wandb/settings
2022-03-13 21:31:20,114 INFO    MainThread:37301 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-13 21:31:20,114 INFO    MainThread:37301 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'Mukund/RE_improved_baseline/meta_learning/train_tacred.py', 'program': 'train_tacred.py'}
2022-03-13 21:31:20,114 INFO    MainThread:37301 [wandb_init.py:_log_setup():386] Logging user logs to /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning/wandb/run-20220313_213120-1bwmqv9l/logs/debug.log
2022-03-13 21:31:20,114 INFO    MainThread:37301 [wandb_init.py:_log_setup():387] Logging internal logs to /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning/wandb/run-20220313_213120-1bwmqv9l/logs/debug-internal.log
2022-03-13 21:31:20,114 INFO    MainThread:37301 [wandb_init.py:init():420] calling init triggers
2022-03-13 21:31:20,115 INFO    MainThread:37301 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-03-13 21:31:20,115 INFO    MainThread:37301 [wandb_init.py:init():471] starting backend
2022-03-13 21:31:20,115 INFO    MainThread:37301 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-13 21:31:20,128 INFO    MainThread:37301 [backend.py:ensure_launched():219] starting backend process...
2022-03-13 21:31:20,138 INFO    MainThread:37301 [backend.py:ensure_launched():225] started backend process with pid: 37452
2022-03-13 21:31:20,139 INFO    MainThread:37301 [wandb_init.py:init():480] backend started and connected
2022-03-13 21:31:20,151 INFO    MainThread:37301 [wandb_init.py:init():550] updated telemetry
2022-03-13 21:31:20,182 INFO    MainThread:37301 [wandb_init.py:init():581] communicating current version
2022-03-13 21:31:21,230 INFO    MainThread:37301 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-13 21:31:21,231 INFO    MainThread:37301 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-13 21:31:21,331 INFO    MainThread:37301 [wandb_init.py:init():624] starting run threads in backend
2022-03-13 21:31:23,128 INFO    MainThread:37301 [wandb_run.py:_console_start():1827] atexit reg
2022-03-13 21:31:23,130 INFO    MainThread:37301 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2022-03-13 21:31:23,132 INFO    MainThread:37301 [wandb_run.py:_redirect():1706] Redirecting console.
2022-03-13 21:31:23,134 INFO    MainThread:37301 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-13 21:31:23,134 INFO    MainThread:37301 [wandb_init.py:init():651] run started, returning control to user process
2022-03-13 22:32:53,327 INFO    MainThread:37301 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 255
2022-03-13 22:32:53,332 INFO    MainThread:37301 [wandb_run.py:_restore():1769] restore
